R/main.R

```diff
@@ -40,7 +40,8 @@ hintr_worker [<queue_id>]"
 main_worker <- function(args = commandArgs(TRUE)) {
   # nocov start
   rrq::rrq_worker(hintr_queue_id(main_worker_args(args)$queue_id, TRUE),
-                  heartbeat_period = 10)
+                  heartbeat_period = 10,
```

Comment:
We might look at another argument for a worker that *only* does calibrations? That would require changes to the deploy too, and I think worth thinking about in separate tickets/PRs

Reply:
Added a ticket https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-2099